android/app/build.gradle

```diff
@@ -71,10 +71,32 @@ android {
     }
 
     buildTypes {
+        debug {
```

Comment:
Ok, zatím můžeme nechat podepsání stejným klíčem. Při vydání se to zřejmě ještě změní a release budeme podepisovat jiným klíčem. Díky za nastavení